src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleArray = sourceString.replace(/^\n+/g, '').split(';');
```

Comment:
The regular expression `/^\n+/g` is used to remove leading newlines, but this might not be necessary if the input is expected to be a single line of CSS styles. Consider removing this unless there is a specific requirement for handling multiline strings.